### ISSUE TITLE
Can't build last version

### ISSUE THREAD

**Issue by btzr-io** [lbryio/lbry-app#582]
## The Issue
I can't build  the latest branch `v16`

### Steps to reproduce
1. Clone branch `v16`
2. run `./build.sh`

### Expected behaviour
`./build.sh` should make a working build.

### Actual behaviour
The build process is closed before installing dependencies:
```
+ false
+ OSNAME=linux
++ node -e 'console.log(require("/home/btzr/prototype/lbry-app/app/package.json").lbrySettings.lbrynetDaemonVersion)'
+ DAEMON_VER=0.16.0rc9
++ node -e 'console.log(require("/home/btzr/prototype/lbry-app/app/package.json").lbrySettings.lbrynetDaemonUrlTemplate)'
+ DAEMON_URL_TEMPLATE=https://github.com/lbryio/lbry/releases/download/vDAEMONVER/lbrynet-daemon-vDAEMONVER-OSNAME.zip
++ echo https://github.com/lbryio/lbry/releases/download/v0.16.0rc9/lbrynet-daemon-v0.16.0rc9-OSNAME.zip
++ sed s/OSNAME/linux/g
+ DAEMON_URL=https://github.com/lbryio/lbry/releases/download/v0.16.0rc9/lbrynet-daemon-v0.16.0rc9-linux.zip
+ DAEMON_VER_PATH=/home/btzr/prototype/lbry-app/build/daemon.ver
+ echo /home/btzr/prototype/lbry-app/build/daemon.ver
/home/btzr/prototype/lbry-app/build/daemon.ver
+ [[ ! -f /home/btzr/prototype/lbry-app/build/daemon.ver ]]
+ wget --quiet https://github.com/lbryio/lbry/releases/download/v0.16.0rc9/lbrynet-daemon-v0.16.0rc9-linux.zip -O /home/btzr/prototype/lbry-app/build/daemon.zip
```
after that I start the app and then get this:
```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'jayson'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/home/btzr/prototype/lbry-app/node_modules/electron/dist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/btzr/prototype/lbry-app/app/main.js:17:16)
    at Object.<anonymous> (/home/btzr/prototype/lbry-app/app/main.js:407:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

**Comment by aayanl** [comment]
related package https://www.npmjs.com/package/jayson

**Comment by btzr-io** [comment]
My problem is that build exit before installing the dependencies from: 
https://github.com/lbryio/lbry-app/blob/master/app/package.json#L11

**Comment by tzarebczan** [comment]
I'm seeing the same issue - stops after the wget on rc9 daemon. When I start the app, it can't seem to find the daemon. Going to try building from scratch again.

**Comment by tzarebczan** [comment]
After fresh re-build, can confirm I am getting the same as @btzr-io

**Comment by kauffj** [comment]
It's pointed at a draft (non-public) RC. This will be fixed when 0.16.0 comes out which should happen soon. If you want to fix your build locally, change rc9 to rc8 in package.json.
